packages/material-ui/src/styles/createTypography.js

```diff
@@ -4,6 +4,24 @@ function round(value) {
   return Math.round(value * 1e5) / 1e5;
 }
 
+let warnedOnce = false;
+
+function roundWithDeprecationWarning(value) {
+  if (process.env.NODE_ENV !== 'production') {
+    if (!warnedOnce) {
+      console.warn(
+        [
+          'Material-UI: The `theme.typography.round` helper is deprecated.',
+          'Head to https://material-ui.com/r/migration-v4/#theme for a migration path.',
```

Comment:
That link is a 404 at the moment. We need to backport it to `master` and push to the `material-ui-docs` origin as well.